docs/js/doc-links.yml

```diff
@@ -226,6 +226,8 @@
           link: /docs/pre-release/howtos/tracing-datadog
         - title: Zipkin
           link: /docs/pre-release/howtos/tracing-zipkin
+    - title: External Filters
+      link: /docs/pre-release/howtos/ext-filters
```

Comment:
The goal of this documentation change was to keep people clicking on "Custom Filters" from being pointed at the `Plugin` `Filter`. I think we should arrange the ToC so that this is under a drop down called "Custom Filters" and have this one and the current `Plugin` `Filter` doc as in that drop down. 

This one should be first. We should also call out why you would need custom filters and why the `External` `Filter` is prefered to the `Plugin` `Filter`.

Reply:
I rearranged the ToC, lines 219-224, I think that matches what you described.
For calling out using `External` over `Plugin` `Filter`s, are you thinking I add a line to the top of the `Plugin` `Filter` doc saying something like "Yeah you can do this but it's a pain in the butt to build and maintain, what you really want is an `External` `Filter`, trust us"?  Would we even provide support if someone was having a problem writing their own `Plugin` `Filter`?